content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -0,0 +1,164 @@
+
+    {{< text syntax=bash snip_id=enable_telemetry >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: telemetry.istio.io/v1alpha1
+    kind: Telemetry
+    metadata:
+      name: otel-demo
+      namespace: istio-system
+    spec:
+      tracing:
+      - providers:
+        - name: otel-tracing
+        randomSamplingPercentage: 100
+        customTags:
+          "my-attribute":
+            literal:
+              value: "default-value"
+    EOF
+    {{< /text >}}
+
+## Deploy the Bookinfo Application
+
+Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
+
+## Generating traces using the Bookinfo sample
+
+1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage`
+    one or more times to generate trace information.
+
+1.  The OpenTelemetry Collector used in the example is configured to export traces to the console.
+    If you used the example Collector config, you can verify traces are arriving by looking
+    at the Collector logs. It should contain something like:
+
+    {{< text syntax=yaml snip_id=none >}}
+    Resource SchemaURL:
+    Resource labels:
+         -> service.name: STRING(productpage.default)
+    ScopeSpans #0
+    ScopeSpans SchemaURL:
+    InstrumentationScope
+    Span #0
+        Trace ID       : 79fb7b59c1c3a518750a5d6dad7cd2d1
+        Parent ID      : 0cf792b061f0ad51
+        ID             : 2dff26f3b4d6d20f
+        Name           : egress reviews:9080
+        Kind           : SPAN_KIND_CLIENT
+        Start time     : 2024-01-30 15:57:58.588041 +0000 UTC
+        End time       : 2024-01-30 15:57:59.451116 +0000 UTC
+        Status code    : STATUS_CODE_UNSET
+        Status message :
+    Attributes:
+         -> node_id: STRING(sidecar~10.244.0.8~productpage-v1-564d4686f-t6s4m.default~default.svc.cluster.local)
+         -> zone: STRING()
+         -> guid:x-request-id: STRING(da543297-0dd6-998b-bd29-fdb184134c8c)
+         -> http.url: STRING(http://reviews:9080/reviews/0)
+         -> http.method: STRING(GET)
+         -> downstream_cluster: STRING(-)
+         -> user_agent: STRING(curl/7.74.0)
+         -> http.protocol: STRING(HTTP/1.1)
+         -> peer.address: STRING(10.244.0.8)
+         -> request_size: STRING(0)
+         -> response_size: STRING(441)
+         -> component: STRING(proxy)
+         -> upstream_cluster: STRING(outbound|9080||reviews.default.svc.cluster.local)
+         -> upstream_cluster.name: STRING(outbound|9080||reviews.default.svc.cluster.local)
+         -> http.status_code: STRING(200)
+         -> response_flags: STRING(-)
+         -> istio.namespace: STRING(default)
+         -> istio.canonical_service: STRING(productpage)
+         -> istio.mesh_id: STRING(cluster.local)
+         -> istio.canonical_revision: STRING(v1)
+         -> istio.cluster_id: STRING(Kubernetes)
+         -> my-attribute: STRING(default-value)
+    {{< /text >}}
+
+## Cleanup
+
+1.  Remove the telemetry
```

Comment:
```suggestion
1.  Remove the Telemetry resource:
```

Reply:
Fixed in [978edaf](https://github.com/istio/istio.io/pull/14566/commits/978edafcdfbeb9bccd1c88174a522964b3576411)